### PR TITLE
Absolute Links for Mobile Library Invite

### PR DESCRIPTION
**Pull request by Colin-Lane**
@stkem

### CODE REVIEW

**Comment by stkem** [comment]
:+1: 
+ @jaredpetker you don't need to do anything on your end for this, right? (See title)

**Comment by Colin-Lane** [comment]
Forgot to change branches before pushing this code. Both are still pretty minor.
@stkem 
I can close this pull and re-open two others with the commits separate, or just leave it how it is.

File: kifi-backend/modules/shoebox/conf/shoeboxService.routes [hunk 1]
```diff
@@ -224,6 +224,7 @@ POST    /m/1/invite                             @com.keepit.controllers.mobile.M
 
 # Profile Page
 GET     /m/1/user/:username/profile             @com.keepit.controllers.mobile.MobileUserProfileController.profile(username)
+GET     /m/1/user/:name/connections             @com.keepit.controllers.mobile.MobileUserProfileController.getProfileConnections(name: Username, limit: Int)
```
Comment (stkem) [inline kifi-backend/modules/shoebox/conf/shoeboxService.routes]: `:username`? to follow the convention in the other routes?

**Comment by stkem** [comment]
Nah, not worth the split effort for this. It's fine like this.

**Pull request merged**